src/plugins/PluginsWrapper.ts

```diff
@@ -0,0 +1,19 @@
+import {InitializationOptions, MonitoredPlugin, OnFailureOptions, OnStartOptions, OnSuccessOptions} from './types';
+
+export class PluginsWrapper implements Omit<MonitoredPlugin, 'initialize'> {
+    constructor(private readonly plugins: MonitoredPlugin[], opts: InitializationOptions) {
+        plugins.forEach((p) => p.initialize?.(opts));
+    }
+
+    onStart(opts: OnStartOptions): void {
+        this.plugins.forEach((p) => p.onStart(opts));
+    }
+
+    onSuccess(opts: OnSuccessOptions): void {
+        this.plugins.forEach((p) => p.onSuccess(opts));
+    }
+
+    onFailure(opts: OnFailureOptions): void {
+        this.plugins.forEach((p) => p.onFailure(opts));
+    }
```

Comment:
Do we need to support multiple plugins at the same time?

Reply:
Yes certainly. We intend for `monitored` to be agnostic from one tool or another, thereby allowing us to support multiple plugins.